docs/docs/docs/developer-resources/testing.md

```
@@ -59,6 +59,64 @@
+
+## **Purpose**  
+The `createMockGraphQLContext` function provides a **fully mocked GraphQL context** for unit and integration testing of GraphQL resolvers. It ensures that resolvers can be tested **without needing a real database, MinIO storage, or authentication service** and works as centralised mocking mechanism.
+
+## **Usage**  
+
+### **Importing the Mock Context**  
+```ts
+import { createMockGraphQLContext } from "test/MockContext/mockGraphQLContext";
+```
+
+### **Creating a Mock Context**  
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)